tools/ts-generator/types/functions.ts

```diff
@@ -14,7 +14,7 @@ import {
   RecordBatch,
   RecordBatchHeader,
 } from "../../domain/generatedRpc/generatedClasses";
-import { calculate } from "fast-crc32c";
+import { calculate } from  "fast-crc32c/impls/js_crc32c";
```

Comment:
did you format this, why the extra spaces? is that from the formatter automatically or bad commit?

Reply:
Yeah just double checked, formatting shows no changes, and the linting step passed. I think theres an issue though because when i run `prettier` it formats 1 file (errors.ts) however when i check that file in the linting step fails. We may need to revisit js linting

Reply:
@andresaristizabal can you help debug this?

Reply:
it doesn't make sense to me that there is an extra space.